internal/plugins/plugins.go

```diff
@@ -16,4 +16,4 @@ package plugins
 
 // DefaultNameQualifier is the Operator SDK plugin name suffix. Appending
 // this suffix to a short name, ex. "go", makes it fully qualified.
-const DefaultNameQualifier = ".operator-sdk.io"
+const DefaultNameQualifier = ".sdk.operatorframework.io"
```

Comment:
Perhaps just 
```suggestion
const DefaultNameQualifier = ".operatorframework.io"
```
since these are operator-framework, and not necessarily SDK, plugins?

Reply:
I see what you're saying. Would that be too generic though? `go.operatorframework.io` seems a little ambiguous.

Reply:
How about `go.plugin.operatorframework.io`.

Reply:
Would we use the same domain for the annotations, too?

@fabianvf any preference?

Reply:
Resolved, I'm ok with `*.sdk.operatorframework.io`.